Chapters/3-1-Chapter15.md

```diff
@@ -0,0 +1,241 @@
+
+_What is this thing?_
+
+She faintly heard some noise coming from it, and hesitantly put it up to her ear. The sound was static at first, but she continued to listen closer as the observers taught her to do with the projectors at the observatory, and through the static she began to make out the tones and rhythms of a voice.
+
+Linyu startled her from a trance-like fixation, standing in front of her with Liffa's robes and hae own wet clothes in hae hands. Thi knew where the listener had gone, and carried a look of consternation. Thi fervidly felt for Melna's hands amidst her confusion and pulled the listener from them, taking no pause to hide it away in hae damp, dirty clothes.
+
+She responded, "I'm sorry, I should have asked first."
+
+Linyu whispered to haeself, "Mom."
+
+"Your mother gave it to you?"
+
+Thi nodded hesistantly, and clutched to it tighter.
+
+"Where is thi now?"
+
+Linyu stared into the void of reality between hae and Melna with eyes like dams holding back a flood of doubt.
+
+"I'm sorry, we don't have to talk about that right now," Melna said, but Linyu had already turned away and started stumbling back into the washroom. A thump rang across the room as Linyu slammed the door behind hae and sat against it to block its movement. She picked up the half-emptied water glass from the table in the great room and and reassumed her post at the door. Melna didn't try to open the door or talk to hae – when Liffa's mother had died, she consoled hae through the same thing and learned how important it was to hold a space for grief. It shook her to imagine feeling such a heavy loss while recovering from the brink of death; such that she had forgotten all of her own stresses and focused on the well-being of this one kaln.
+
+"Linyu, may I come in?"
+
+There was no response.
+
+"Knock once if you want me to leave. Twice if you'd rather I stay here."
+
+After a long stillness there came a shuffling and then a knocking against the door. _Tap...Tap._
+
+Slightly relieved, she responded, "Okay. I'll stay then."
+
+She decided to be deliberate. Another prying question might push Linyu to leave, and it had become far too cold for anyone to be outside for the night.
+
+"I promise that I won't ask any more questions. And I won't steal your device. I just have some water for you here by the door."
+
+Thi slowly creaked the door open just enough to listen for Melna and reach for the glass which Melna gave eagerly. Thi kept the door cracked like this, sipping on water and listening vigilantly for any changes or movements. A crack of light shone through and Melna could see the texture of Liffa's robes clothing Linyu's shoulders.
+
+"You know, my jundmaman..." she began, fighting off her own tears, "Thi used to run to the washroom whenever thi was upset too. I never did a great job comforting hae. The other one was always better at it."
```

Comment:
> Thi used to run to the washroom whenever thi was upset too.
ah i see....

> The other one was always better at it.
Feels unnatural. "Kipnu was always better at it," sounds more realistic even if Melna is aware Linyu might not know who Kipnu is..